site/static/admin/config.yml

```diff
@@ -8,6 +8,53 @@ media_folder: static/assets
 public_folder: assets
 
 collections:
+  - name: articles
+    label: Articles
+    label_singular: 'Article'
+    folder: content/articles
+    create: true
+    fields:
+      - { name: title, label: Title }
+      - { name: date, label: Publish Date, widget: date }
+      - { name: description, label: Description }
+      - { name: body, label: Main Body, widget: markdown }
+      - { label: "Repo",
+          name: "repo",
+          widget: "relation",
+          collection: "repos",
+          searchFields: ["title"],
+          valueField: "title",
+          displayFields: ["title"]}
+      - { name: priority, label: Priority, widget: "number", default: 0,
+          hint: "Controls the order in which the repo appears in the parent repo, starting from 0, lower = higher"  }
+      - { name: role, label: Role, required: false }
+      - { name: author, label: Author, required: false }
+      - { name: reviewedby, label: Reviewed By, required: false }
+      - { name: signedby, label: Signed Off, required: false }
+      - { name: tags, label: Tags, widget: "list", required: false,
+          hint: "divide tags up using commas, e.g:  emergency, test, contact " }
+
+
+  - name: repos
```

Comment:
Can we change the name of repos, to a word aligned with what a content editor would understand?  Perhaps "Grouping" ... maybe "MenuItem", "Category" ... chat with Philip and pick a content editor aligned word.